compiler/passes/buildDefaultFunctions.cpp

```diff
@@ -641,6 +643,41 @@ static void build_record_inequality_function(AggregateType* ct) {
   normalize(fn);
 }
 
+
+static void build_enum_size_function(EnumType* et) {
+  if (function_exists("size", 1, et))
+    return;
+  // Build a function that returns the length of the enum specified
+  FnSymbol* fn = new FnSymbol("size");
+  fn->addFlag(FLAG_COMPILER_GENERATED);
+  fn->addFlag(FLAG_NO_PARENS);
+  fn->addFlag(FLAG_METHOD);
+  fn->insertFormalAtTail(new ArgSymbol(INTENT_BLANK, "_mt", dtMethodToken));
+  
+  ArgSymbol* _this = new ArgSymbol(INTENT_BLANK, "this", dtAny);//et);
```

Comment:
It probably should say 'size' not 'length', but it was an intentional comment to mirror the commenting style of the other build_enum_thing functions